.github/workflows/deploy_main.yml

```diff
@@ -8,6 +8,10 @@ on:
     branches: [main]
     paths:
       - "client/**/*"
+env: 
```

Comment:
Up on line 7, it looks like this runs when a PR is closed (` types: [closed]`). Should this be `merged` instead or additionally (if that's an option)? I don't see any code in this action that suggests that it only runs on merge (as opposed to a PR being closed without merging), so I want to confirm that this is only being run on merge... 

Now that I think about it though, while the deploy stuff should only happen on merge of the PR, the stale artifact cleanup should happen on any type of close, regardless of merge.

Reply:
While this makes sense to me, checking the PR types, I don't think 'merged' is an option?

https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request

Reply:
Yeah, @switzersc-usds, `closed` is the right action, since it's meant to be kicked off when the review is finished and the PR is closed, which signals the final merge.